### PR TITLE
Goal page is finished ✨

### PR DESCRIPTION
**Pull request by farahccino**
Please test this:
- [x] go on the plus page and add random habits with any Häufigkeit
- [x] they should show up on the goals page
- [x] by clicking on the habit an editing modal should show up
- [x] you can edit or delete your habit or just return to the previous page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,11 +6,13 @@
     "@testing-library/jest-dom": "^5.12.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "moment": "^2.29.1",
```
Comment (anitaschiller) [inline package.json]: Do you really need this module? If not, pls remove :)

File: package.json [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
+    "react-table": "^7.7.0",
```
Comment (anitaschiller) [inline package.json]: Not needed anymore, you can delete it!

File: package.json [hunk 3]
```diff
@@ -40,6 +42,8 @@
     ]
   },
   "devDependencies": {
+    "@types/react-table": "^7.7.1",
```
Comment (anitaschiller) [inline package.json]: Pls remove!

File: src/components/BasicTable.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+
+export default function BasicTable({ habits, onEditHabit }) {
+  console.log(habits, "habits");
```
Comment (anitaschiller) [inline src/components/BasicTable.js]: Please remove all console.logs before merging!

File: src/components/BasicTable.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+          <th>Häufigkeit</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {habits.map((habit) => {
+          return (
+            <tr onClick={() => onEditHabit(habit)}>
+              <td>{habit.ziel}</td>
+              <td>{habit.häufigkeit}</td>
```
Comment (anitaschiller) [inline src/components/BasicTable.js]: Try to stick to English variable and property names :)

File: src/components/FooterNavigation.js [hunk 6]
```diff
@@ -1,37 +1,33 @@
...
-        </NavLink>
-        <NavLink to="/add-goal" className="link">
-         <img src={plus} alt='Adds a new entry' width="64" height="64" />
-        </NavLink>
-        <NavLink to="/goals" className="link">
-         <img src={goals} alt='Goals page' width="64" height="64" />
-        </NavLink>
-     </NavWrapper>
-    )
+      <NavLink to="/today" className="link">
```
Comment (anitaschiller) [inline src/components/FooterNavigation.js]: Is your class `link` used anywhere? If not, pls remove :)

File: src/index.js [hunk 7]
```diff
@@ -1,19 +1,15 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import { GlobalStyle } from './GlobalStyle';
+import React from "react";
```
Comment (anitaschiller) [inline src/index.js]: Try to stay consistent with single or double quotes throughout your code base. Configure prettier in VS Code or add a .prettierrc file :)

File: src/pages/Home.js [hunk 8]
```diff
@@ -1,12 +1,13 @@
 import { useState } from "react";
 import { NavLink } from "react-router-dom";
+import moment from "moment";
```
Comment (anitaschiller) [inline src/pages/Home.js]: Moment is a huge package and adds a lot of weight to your project.
Please use a lightweight database like date-fns.

File: src/pages/Goals.js [hunk 9]
```diff
@@ -1,12 +1,66 @@
...
+import FooterNavigation from "../components/FooterNavigation";
+import BasicTable from "../components/BasicTable";
+import HabitForm from "../components/HabitForm";
+
+export default function Goals({
+  habits,
+  setHabits,
+  id,
+  onSetHabitToEdit,
+  setActivePage,
```
Comment (anitaschiller) [inline src/pages/Goals.js]: Check if these are still needed!

**Review by anitaschiller — CHANGES_REQUESTED** [review]
Looks good, but a few optimizations are pending :)

**Comment by farahccino** [comment]
Adjusted everything, thank you so much :)

**Pull request merged**